### PR TITLE
Change test_erros to pytest 

### LINKED ISSUE #6503: Switch tests to py.test from nose
nose is unmaintained and recommends to use nose2, unittest or pytest

pytest has fixture based dependency injection, and parametrization, and is generally better

### PR DESCRIPTION
**Pull request by diru1100**
Fixes #6503

#### Short description of what this resolves:
Converts test code to pytest standard

#### Changes proposed in this pull request:

Change in test code

#### Checklist

- [x ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [x ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: tests/all/unit/api/helpers/test_errors.py [hunk 1]
```diff
@@ -9,43 +8,32 @@
...
-        bad_request_error = BadRequestError({'source': ''}, 'Request cannot be served')
-        self.assertEqual(bad_request_error.status, 400)
+"""Test for error responses"""
+
+
+def test_error_response_dict_details():
+    """To test details in the form of dict"""
+    error_response = ErrorResponse(source="test source", detail="test detail")
+    expected_dict = {
+        'status': error_response.status,
```
[the remark this answers is not in the document]
Comment (diru1100) [reply tests/all/unit/api/helpers/test_errors.py]: black command was run pre-commit

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- tests/all/unit/api/helpers/test_errors.py  5
- tests/all/unit/api/validation/test_createdat.py  5
- app/factories/user_token_blacklist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955&bid=16317085)

File: tests/all/unit/api/helpers/test_errors.py [hunk 2]
```diff
@@ -9,43 +8,32 @@
...
-        # UnprocessableEntity Error
-        unprocessable_entity_error = UnprocessableEntityError(
-            {'source': ''}, 'Entity cannot be processed'
-        )
-        self.assertEqual(unprocessable_entity_error.status, 422)
-
-        # Bad Request Error
-        bad_request_error = BadRequestError({'source': ''}, 'Request cannot be served')
-        self.assertEqual(bad_request_error.status, 400)
+"""Test for error responses"""
```
Comment (niranjan94) [inline tests/all/unit/api/helpers/test_errors.py]: Codacy Issue found: [String statement has no effect](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955)

File: tests/all/unit/api/helpers/test_errors.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
-from unittest import TestCase
-
+import pytest
```
Comment (niranjan94) [inline tests/all/unit/api/helpers/test_errors.py]: Codacy Issue found: [Unused import pytest](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955)

Comment (niranjan94) [inline tests/all/unit/api/helpers/test_errors.py]: Codacy Issue found: ['pytest' imported but unused](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/unit/api/validation/test_createdat.py  5
- tests/all/unit/api/helpers/test_errors.py  5
- app/factories/user_token_blacklist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955&bid=16317085)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/unit/api/helpers/test_errors.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4933955&bid=16317085)

**Pull request closed without merging**